src/main/scala-2.11/markets/clearing/ClearingMechanismLike.scala

```diff
@@ -15,13 +15,45 @@ limitations under the License.
 */
 package markets.clearing
 
-import markets.BaseActor
+import akka.actor.ActorRef
+import akka.agent.Agent
+
+import markets.{BaseActor, Cancel, Canceled}
 import markets.clearing.engines.MatchingEngine
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Imports should be sorted alphabetically](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1560525559/issues/source?bid=2921361&fileBranchId=3019109#l22)